haskell/toolchain.bzl

```
@@ -80,7 +80,11 @@ while IFS= read -r line; do compile_flags+=("$line"); done < %s
 while IFS= read -r line; do extra_args+=("$line"); done < %s
 while IFS= read -r line; do param_file_args+=("$line"); done < %s
 
-"${compile_flags[@]}" "${extra_args[@]}" ${param_file_args+"${param_file_args[@]}"}
+# XXX Workaround https://gitlab.haskell.org/ghc/ghc/merge_requests/1308.
+output=$("${compile_flags[@]}" "${extra_args[@]}" ${param_file_args+"${param_file_args[@]}"} 2>&1)
+rc=$?
+echo "$output" | while IFS= read -r line; do [[ $line =~ ^Loaded ]] || echo "$line"; done
```

```suggestion
echo "$output" | while IFS= read -r line; do [[ $line =~ ^Loaded ]] || echo "$line" >&2; done
```
To keep `bazel build`'s stdout reasonably parseable.